src/releng_docs/projects/pulselistener.rst

```diff
@@ -52,7 +52,7 @@ Manager.
 .. _Release Management: https://wiki.mozilla.org/Release_Management
 .. _Pulse: https://wiki.mozilla.org/Auto-tools/Projects/Pulse
 .. _Heroku: https://www.heroku.com/
-.. _shipit code coverage production hook: https://tools.taskcluster.net/hooks/project-releng/services-production-shipit-code-coverage
+.. _shipit code coverage production hook: https://tools.taskcluster.net/hooks/project-releng/services-production-codecoverage
```

Comment:
We should fix this hook and the static analysis hook too.

Reply:
The code coverage one is: https://tools.taskcluster.net/hooks/project-releng/services-production-codecoverage%2Fbot.